src/OpdsBuilder.php

```diff
@@ -55,7 +60,7 @@ public function buildFromCategory( $categoryTitle ) {
 			'gcmlimit' => '100',
 		] );
 		if ( !array_key_exists( 'query', $response ) ) {
-			throw new NotFoundHttpException( "Category not found: $categoryTitle" );
+			throw new Exception( "Category not found: $categoryTitle" );
```

Comment:
Did we want this to be translatable, too? I think it's okay if we don't (probably a rare error), but as written the changes to this file don't have any user-facing effect, do they?

Reply:
That's right: this is only used in a Command, and none of those are translated.